examples/Cpp/PrimitiveRestart/Example.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ * Example.cpp (Example_PrimitiveRestart)
+ *
+ * This file is part of the "LLGL" project (Copyright (c) 2015-2019 by Lukas Hermanns)
+ * See "LICENSE.txt" for license information.
+ */
+
+#include <ExampleBase.h>
+#include <vector>
+
+class Example_PrimitiveRestart : public ExampleBase
+{
+    LLGL::ShaderProgram*    shaderProgram       = nullptr;
+    LLGL::PipelineState*    pipeline            = nullptr;
+    LLGL::Buffer*           vertexBuffer        = nullptr;
+    LLGL::Buffer*           indexBuffer16       = nullptr;
+    LLGL::Buffer*           indexBuffer32       = nullptr;
+
+    size_t                  indexCount16        = 0;
+    size_t                  indexCount32        = 0;
+
+public:
+
+    Example_PrimitiveRestart() :
+    ExampleBase { L"LLGL Example: PrimitiveRestart" }
+    {
+        auto vertexFormats = CreateBuffers();
+        shaderProgram = LoadStandardShaderProgram(vertexFormats);
+        CreatePipeline();
+    }
+
+private:
+
+    // Vertex data structure
+    struct Vertex
+    {
+        Gs::Vector2f        position;
+        LLGL::ColorRGBAub   color;
+    };
+
+    template <typename T>
+    static void addSquare(float centerX, float centerY, float size, std::vector<Vertex>& vertices, std::vector<T>& indices)
```

Comment:
Please don't use _lower camel case_ for functions in this project. I wrote a coding style guideline for this project a while back: [LLGL Coding Conventions](https://github.com/LukasBanana/LLGL/blob/master/docu/CodingConventions/Coding%20Conventions%20for%20LLGL.pdf). The rest looks very good 👍